addon/templates/components/frost-modal-dialog.hbs

```diff
@@ -59,7 +59,16 @@
   {{! END-SNIPPET }}
 
   {{#if params.footer}}
-    <div class="frost-modal-dialog-footer-text">{{params.footer}}</div>
+    {{#if (type-of params.footer 'object')}}
+      <div class="frost-modal-dialog-footer-object"
+        data-test={{hook (concat hook '-object')}}>
+        {{component params.footer}}
```

Comment:
Any way to remove that complaints?  It lead me to believe that it wasn't passed properly or wasn't supported when I tried it.